infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```
@@ -235,6 +235,9 @@ param parTags object = {}
 @sys.description('Set Parameter to true to Opt-out of deployment telemetry.')
 param parTelemetryOptOut bool = false
 
+@sys.description('Define outbound destination ports or ranges for SshRDPOutbound. Default:[22,3389]')
+param parBastionOutboundSshRDPPort array = ['22','3389']
```

```suggestion
param parBastionOutboundSshRdpPorts array = ['22','3389']
```
casing and making plural